packages/features/ee/workflows/lib/actionHelperFunctions.ts

```diff
@@ -95,3 +109,28 @@ export function getWhatsappTemplateForAction(
   const templateFunction = getWhatsappTemplateFunction(template);
   return templateFunction(true, locale, action, timeFormat);
 }
+
+export function getTemplateBodyForAction({
```

Comment:
This function is used in `WorkflowStepContainer` when changing the template or action. Also used in tasker to grab the default template